game/src/main/scala/io/aigar/model/TeamRepository.scala

```diff
@@ -25,6 +26,12 @@ class TeamRepository(databaseName: Option[String]) {
     TeamDAO.findTeamBySecret(db, teamSecret)
   }
 
+  def updateScore(scoreMessage: ScoreMessage): Unit ={
```

Comment:
I'm not sure the repo should be aware of `ScoreMessage`, what do you think?

Also, those query are not atomic. We could use an transaction. But do we put the effort since, theoretically, there is only one thread doing this query?